### PR TITLE
Demonstrate passing "max_tokens" param

### PR DESCRIPTION
**Pull request by mkhludnev**
context https://github.com/triton-inference-server/server/issues/6864

### CODE REVIEW

File: samples/client.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ async def process_stream(self, prompts, sampling_parameters):
                     self._results_dict[result.get_response().id].append(i)
 
     async def run(self):
-        sampling_parameters = {"temperature": "0.1", "top_p": "0.95"}
+        sampling_parameters = {"temperature": "0.1", "top_p": "0.95", "max_tokens":"100"}
```
Comment (oandreeva-nv) [inline samples/client.py:93]: Thanks for this PR! Our `client.py` serves as a sample client, thus maybe it makes sense to put some comments above and not an actual code

Comment (oandreeva-nv) [inline samples/client.py:93]:
```suggestion
        # Sampling parameters for text generation, required by vLLM model,
        # including `temperature`, `top_p`, top_k`, `max_tokens`, `early_stopping`.
        # Full list available at:
        # https://github.com/vllmproject/vllm/blob/5255d99dc595f9ae7647842242d6542aa4145a4f/vllm/sampling_params.py#L23
        sampling_parameters = {"temperature": "0.1", "top_p": "0.95", "max_tokens":"100"}
```

Comment (oandreeva-nv) [reply samples/client.py:93]: Something like this /\

Comment (nnshah1) [reply samples/client.py:97]: @oandreeva-nv We can add documentation to the README for the currently supported sampling parameters and/or provide command line arguments to set them in the sample client. Realized we don't currently document the list of supported sampling parameters -

Comment (oandreeva-nv) [reply samples/client.py:97]: `supported sampling parameters` are actively changing. I would avoid specifying list in README ATM

**Comment by oandreeva-nv** [comment]
@mkhludnev Could you please remind me if you've signed a CLA?

**Comment by oandreeva-nv** [comment]
Started pipeline: 13124488

File: samples/client.py [hunk 2]
```diff
@@ -90,7 +90,11 @@ async def process_stream(self, prompts, sampling_parameters):
                     self._results_dict[result.get_response().id].append(i)
 
     async def run(self):
-        sampling_parameters = {"temperature": "0.1", "top_p": "0.95"}
+        # Sampling parameters for text generation, required by vLLM model,
+        # including `temperature`, `top_p`, top_k`, `max_tokens`, `early_stopping`.
+        # Full list available at:
+        # https://github.com/vllmproject/vllm/blob/5255d99dc595f9ae7647842242d6542aa4145a4f/vllm/sampling_params.py#L23
+        sampling_parameters = {"temperature": "0.1", "top_p": "0.95", "max_tokens":"100"}
```
Comment (oandreeva-nv) [inline samples/client.py:97]:
```suggestion
        sampling_parameters = {
            "temperature": "0.1",
            "top_p": "0.95",
            "max_tokens": "100",
        }
```

Comment (oandreeva-nv) [reply samples/client.py:97]: our linter fails on this https://github.com/triton-inference-server/vllm_backend/actions/runs/7926898452/job/22046723696?pr=34#step:4:151

**Comment by mkhludnev** [comment]
> @mkhludnev Could you please remind me if you've signed a CLA?

Just did it. Sent.

**Review by oandreeva-nv — APPROVED** [review]

**Comment by nv-kmcgill53** [comment]
Hi @mkhludnev, we have accepted your CLA. Thank you for contributing to the Triton project!

File: samples/client.py [hunk 3]
```diff
@@ -90,7 +90,15 @@ async def process_stream(self, prompts, sampling_parameters):
                     self._results_dict[result.get_response().id].append(i)
 
     async def run(self):
-        sampling_parameters = {"temperature": "0.1", "top_p": "0.95"}
+        # Sampling parameters for text generation, required by vLLM model,
```
Comment (nnshah1) [inline samples/client.py:93]:
```suggestion
        # Sampling parameters for text generation
```

Comment (nnshah1) [reply samples/client.py:93]: small tweak to comment -

**Review by oandreeva-nv — DISMISSED** [review]

**Comment by oandreeva-nv** [comment]
@mkhludnev feel free to merge it

**Comment by mkhludnev** [comment]
> @mkhludnev feel free to merge it

Thanks for the proposal, @oandreeva-nv, but I don't think I'm a committer here. Just resolved a conflict.

**Review by oandreeva-nv — APPROVED** [review]

**Pull request merged**